lib/translator/index.js

```diff
@@ -227,11 +207,7 @@ Translator.prototype._compileTemplate = function (key) {
     try {
         return formatter.compile(language.translations[key]);
     } catch (err) {
-        if (err.name === 'SyntaxError') {
-            this.logger.error(`Language File Syntax Error: ${err.message} for key "${key}"`, err.expected);
-
-            return () => '';
```

Comment:
Before we were returning, and now we're throwing.

Does this mean that on a production store, the page would return an error such as 500, instead of the helper simply failing quietly and returning an empty string?